docs/v1/use-cases.md

```diff
@@ -1,9 +1,26 @@
 ## V1 Use Cases
 
-This document contains the complete list of accepted use-cases for the v1 version of `service-catalog`.
+This document contains the complete list of accepted use-cases for the v1
+version of `service-catalog`.
 
-## Use Cases
+## Use cases
 
-The v1 service-catalog supports the following use cases:
+1.  Sharing services:
+    1.  As a SaaS provider that already runs a Service Broker, I want users of
+        Kubernetes to be able to use my service via the Service Broker API, so
+        that I can grow my user base to include users of Kubernetes
 
-1. <TBD>
+### SaaS providers with existing Service Brokers
+
+There are numerous SaaS providers that already operate Service Brokers today.
+It should be possible for the operator of an existing Service Broker to
+publish their services into the catalog and have them consumed by users of
+Kubernetes.  This offers a new set of users to the service operator and offers
+a wide variety of SaaS products to users of Kubernetes.
+
+When a Kubernetes user binds to a service of this type, the catalog creates
+the following resources in their namespace:
+
+1.  A Kubernetes `Service` instance, to be a stable network endpoint that makes
```

Comment:
If we're talking about blackbox services, I don't think we're talking about new resources being created within the user's namespace. The service owner will manage the resources in their own hosting environment - which may or may not be **this** kubernetes instance.  When a user "binds" to a service all they get are the creds of how to talk to it. The point of a blackbox service is that the user doesn't see, or have access, to the service instance itself.